README.md

```diff
@@ -11,3 +11,5 @@ By the end of the lesson, you will be able to identify which workflow best suits
 your project and style.
 
 Let's begin with a simple [Markdown primer](./markdown-primer.md)!
+
+Hi, Max! Here is a pull request.
```

Comment:
Hi Dena!
This addition looks fantastic! Thx so much for keeping this playground so cool!